spec/jsSpecs/ares-device.spec.js

```diff
@@ -179,13 +218,21 @@ describe(aresCmd, function() {
 });
 
 describe(aresCmd + ' --resource-monitor(-r)', function() {
```

Comment:
how about " #204~217" move to #123 and merge resource-monitor case?

Reply:
Done.